### DISCUSSION TITLE
Muitos processos, ajudam ou atrapalham?!

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#771] [Discussão]
Recentemente vi Issues sobre hábitos mais saudáveis, hábitos de estudo etc, ando lendo muito sobre e vejo que hoje existe vários processos e metodologias para se organizar no dia a dia, mas es a questão, adicionar muitos processos ao dia a dia ajudam ou atrapalham?! Qual a opinião de vocês sobre isso?! :thinking:

**Comment by yanmagale** [comment]
@Cleysonlb  boa pergunta.

Na issue #679 a galera falou bastante de diversos tipos de métodos e práticas.

Acho que é interessante sim você conhecer outras metodologias e técnicas, mas penso que o mais importante é você descobrir qual o melhor método para **_você_.** Ter muitos métodos podem sim lhe atrapalhar, mas acredito que isto depende muito de pessoa para a pessoa, envolvendo diversos fatores, principalmente o momento de vida de cada um.

Eu particularmente gosto muito de ler artigos como estes que você citou. Quando vejo algo novo, procuro adaptar a minha rotina e aos meus métodos. Se funcionar bem ou melhor do que as coisas que eu ja faço, eu procuro me adaptar a este novo estilo, caso contrário, eu não o levo em consideração **_naquele momento_**. Caso nenhum destes métodos mais tradicionais não funcionem tanto com você, não hesite em criar o _seu_ próprio modelo, afinal de contas, apenas **_você_** poderá dizer o que melhor funciona para você.


Espero ter ajudado.
